app/code/RLTSquare/Ccq/etc/module.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="RLTSquare_Unit2"/>
+    <sequence>
+        <module name="Magento_MessageQueue"/>
+        <module name="Magento_Mysqlmq"/>
```

Comment:
Unnecessary/inappropriate code - good intention, not really necessary ... as our module should not be aware of what type queue we will be using (mysql, rabbitmq, etc...). Possibly add dependency to Magento_MessageQueue.